File: programmers 문제 모음/연습문제/가운데글자가져오기.py
def solution(s):
    i = int(len(s)//2)
    if len(s) % 2: # 문자열 길이가 홀수인 경우
        answer = s[i]
    else: # 문자열 길이가 짝수인 경우
        answer = s[i-1:i+1]    
    return answer

File: programmers/연습문제/완전탐색/가운데 글자 가져오기.js
function solution(s) {
  let half = parseInt(s.length / 2);
  if (s.length % 2) { // 글자 길이가 홀수인 경우
    return s[half];
  } else { // 글자 길이가 짝수인 경우
    return s.slice(half - 1, half + 1);
  }
}
